hilti/toolchain/include/ast/expression.h

```diff
@@ -46,6 +46,7 @@ using expression::detail::to_node;
 /** Constructs an AST node from any class implementing the `Expression` interface. */
 template<typename T, typename std::enable_if_t<std::is_base_of<trait::isExpression, T>::value>* = nullptr>
 inline Node to_node(T t) {
+    // NOLINTNEXTLINE(clang-analyzer-cplusplus.NewDeleteLeaks)
```

Comment:
There are quite a few of these, looks like we should just disable this checker? Some quick Googling shows quite a few reports about false positives. We also have ASAN to catch such stuff.

Reply:
I disabled this check for now and removed the adjustments it triggered.